src/main/java/com/amplitude/Amplitude.java

```diff
@@ -50,8 +52,10 @@ public void logEvent(String name) {
 
     public void logEventWithProps(String eventName, JSONObject eventProps) {
         long time = System.currentTimeMillis();
-        Event event = new Event(eventName, eventProps, userProperties, "", Constants.SDK_VERSION,
-                lastEventId, sessionId, userId, deviceId, time);
+        Event event = new Event(eventName);
```

Comment:
This is temporary, right?

Reply:
I have already cleaned up this in the class. I'll fix the merge conflict when this is merged.